1.8/usage/haproxy.md

```diff
@@ -0,0 +1,77 @@
+---
+post_title: Configuring HTTP Proxy in Front of DC/OS
+nav_title: Configure HTTP Proxy
+menu_order: 61
+---
+
+You can configure a secure communication channel between HAProxy and the DC/OS Admin Router, using a custom hostname and port.
```

Comment:
The original proposed sentence is a bit ambiguous. I'd propose `You can put an HTTP proxy in front of the DC/OS Admin Router.`